### DISCUSSION TITLE
[Bug]: Pimcore Bricks are not working

### DISCUSSION THREAD

**Discussion by NadineKaul** [pimcore/pimcore#13202] [Q&A / Support]
### Expected behavior

Default Pimcore Bricks are working with demo tutorial.

### Actual behavior

Pimcore Bricks can not be found and used in Backend and Frontend since some of the latest updates. 

### Use demo tutorial from Pimcore Academy. 
Using **xampp** with php >= 8

- Tools -> Bundles and Bricks  are not visible in list.  
- Documents -> edit -> shows an error:
  Areabrick wysiwyg-with-images is not registered File: C:\xampp\htdocs\my- 
  project\vendor\pimcore\pimcore\lib\Extension\Document\Areabrick\AreabrickManager.php Line: 129
- Frontend shows the same error: 
  Areabrick hero-teaser is not registered File: C:\xampp\htdocs\my- 
  project\vendor\pimcore\pimcore\lib\Extension\Document\Areabrick\AreabrickManager.php Line: 129

### I tried to fix it with: 
pimcore:build:classes 
pimcore:cache:clear

### I checked the system requirements:
 bin/console pimcore:system:requirements:check
 ------------------------------ ---------
  PHP
 ------------------------------ ---------
  memory_limit (in php.ini)      ok
  PDO MySQL                      ok
  Mysqli                         ok
  iconv                          ok
  Document Object Model (DOM)    ok
  SimpleXML                      ok
  GD                             ok
  EXIF                           ok
  Multibyte String (mbstring)    ok
  File Information (file_info)   ok
  zip                            ok
  zlib / gzip                    ok
  Intl                           ok
  locales-all                    ok
  Imagick                        warning
  APCu                           warning
  OPcache                        ok
  Redis                          warning
  curl                           ok
  WebP (via GD)                  ok
  AVIF (via GD)                  warning
 ------------------------------ ---------

 -------------------------------- ----
  MySQL
 -------------------------------- ----
  InnoDB Support                   ok
  ARCHIVE or MyISAM Support        ok
  Database Charset utf8mb4         ok
  innodb_large_prefix = ON         ok
  innodb_file_format = Barracuda   ok
  innodb_file_per_table = ON       ok
  CREATE TABLE                     ok
  ALTER TABLE                      ok
  Manage Indexes                   ok
  Fulltext Indexes                 ok
  INSERT                           ok
  UPDATE                           ok
  SELECT                           ok
  CREATE VIEW                      ok
  SELECT (from view)               ok
  DELETE                           ok
  SHOW CREATE VIEW                 ok
  SHOW CREATE TABLE                ok
  DROP VIEW                        ok
  DROP TABLE                       ok
  WITH RECURSIVE                   ok
 -------------------------------- ----

 ------------------------------------- ---------
  Filesystem
 ------------------------------------- ---------
  /var (not checked - too many files)   warning
 ------------------------------------- ---------

 ------------------------------------- ---------
  CLI Tools & Applications
 ------------------------------------- ---------
  PHP                                   ok
  Composer                              ok
  FFMPEG                                warning
  Chromium                              warning
  Ghostscript                           warning
  LibreOffice                           warning
  jpegoptim                             warning
  pngquant                              warning
  optipng                               warning
  exiftool                              warning
  timeout - (GNU coreutils)             ok
  pdftotext - (part of poppler-utils)   ok
  facedetect                            warning
  Graphviz                              warning
 ------------------------------------- ---------

**Comment by aashan10** [comment]
Could not reproduce this issue on latest release v10.5.5 @NadineKaul please re-verify that this issue exists!
And also please make sure that you are running the migrations after updating the pimcore version.

**Comment by deepakdhanji** [comment]
@aashan10 I can confirm that this issue happens with fresh installation of v10.5.5 done in accordance with the [Pimcore Installation Guide](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html)

In order to get around the issue, I had to manually register all the Areabricks in services.yaml. Attaching my working services.yaml file for use by any other new-comer like me.
[services.yaml](https://gist.github.com/deepakdhanji/93587f952cced956b8e57722912c01ad)

**Comment by myspulin** [comment]
I have the same issue on fresh installation v10.5.6 on windows by https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html

Frontend show the error: Areabrick wysiwyg is not registered

**Comment by aashan10** [comment]
@deepakdhanji Pimcore does not recommend using XAMPP. I would suggest you to use a linux machine with the required softwares installed. Or if you are developing/running pimcore in windows, please use the provided docker images instead.
